src/components/Vault/VaultStrategy.tsx

```diff
@@ -135,21 +145,23 @@ const VaultStrategyModal: FC<VaultStrategyModalProps> = ({
               liquidity pool. Holders earn fees from users trading in the pool,
               and can also deposit the LP to Curve's gauges to earn CRV
               emissions. This Curve v2 crypto pool contains{" "}
-              {underlyingAssets?.map((address, index) => (
-                <Fragment key={`underlying-asset-${index}`}>
-                  {underlyingAssets.length > 2 &&
-                  index > 0 &&
-                  index !== underlyingAssets.length - 1
-                    ? ", "
-                    : index > 0
-                    ? " and "
-                    : null}
-                  <TokenSymbol
-                    key={`token-symbol-${index}`}
-                    address={address}
-                  />
-                </Fragment>
-              ))}
+              {underlyingAssets?.map(
+                (address: string | undefined, index: number) => (
```

Comment:
Change the type on line 84 to `underlyingAssets: UseCompounderUnderlyingAssetsResult["data"]` and you can remove the types set for address + index here.

Reply:
done